PublicMapping/districtbuilder
# Style thank you page & small UI fixes

## Overview
This PR styles the Thank You page that a user gets dropped into after activating their account. It also fixes an alignment issue for the user dropdown in the HomeScreen landing page and makes focus states better for checkboxes and radio buttons.

### Checklist


### Demo
<img width="1680" alt="Screen Shot 2020-09-03 at 2 08 59 PM" src="https://user-images.githubusercontent.com/5672295/92120976-a3d88a00-edf1-11ea-92c5-9df741e25783.png">

Radio inputs
<img width="877" alt="Screen Shot 2020-09-03 at 12 57 47 PM" src="https://user-images.githubusercontent.com/5672295/92121023-b2bf3c80-edf1-11ea-9f37-b79ddd96f1da.png">

Checkbox inputs
<img width="806" alt="Screen Shot 2020-09-03 at 12 58 35 PM" src="https://user-images.githubusercontent.com/5672295/92121005-acc95b80-edf1-11ea-924a-79516e8097f6.png">

Firefox user dropdown
<img width="573" alt="Screen Shot 2020-09-03 at 2 19 26 PM" src="https://user-images.githubusercontent.com/5672295/92121139-d7b3af80-edf1-11ea-87a5-0395f83356a1.png">


### Notes
- I usually prefer the focus outline to be tighter to the input icon, but it wasn't quick to figure out so I figured that this is at least an improvement over what we had.

## Testing Instructions
- `git pull`
- Create a new account. You'll receive an email to register it.
- In your browser, remove everything after `localhost:3000` and 
- **Be careful not to click the link in your email!** Grab the following from the url in the email: `/activate/[token]`.
- Paste this into your browser's url box.
- View the page, note that it's styled as above.
- Log into the application from Firefox.
- You should be on the home screen. Take a look at the user dropdown in the top right. The caret should be vertically centered.
- Now click the "add a project" button.
- Start adding a project.
- Radio button focus states should look as they do in screenshots above.
- Bonus: replace the inputs here with `Checkbox` components to see how those look when focused.

Closes #321, closes #381
